tests/integrations/mcs/tso/keyspace_group_manager_test.go

```diff
@@ -151,3 +160,71 @@ func request(
 	re.NoError(tsoClient.Send(req))
 	return tsoClient.Recv()
 }
+
+func cleanupKeyspaceGroups(re *require.Assertions, server *tests.TestServer) {
+	for _, group := range handlersutil.MustLoadKeyspaceGroups(re, server, "0", "0") {
+		handlersutil.MustDeleteKeyspaceGroup(re, server, group.ID)
+	}
+}
+
+func (suite *tsoKeyspaceGroupManagerTestSuite) TestTSOKeyspaceGroupSplitElection() {
```

Comment:
Do we need to test if the client can work normally in the split process?

Reply:
I added a test case for the client. However, we may need to wait for #6321 to be merged first to make it work.

Reply:
ok, let's speed up #6321 then. please add review feedbacks to it.